frontend/.gitignore

```diff
@@ -27,3 +27,7 @@ src/locales/scripts/wp-locales.json
 /.npmrc
 /.pnpmfile.cjs
 /pnpm-lock.yaml
+
+# To prevent accidentally adding a hardcoded robots.txt, see
+# /src/server-middleware/robots.js for the robots.txt file.
+src/static/robots.txt
```

Comment:
:facepalm: Sorry I missed this! I thought I had looked for an existing robots thing in the frontend, but I must only have looked in nuxt.config and not seen anything there. Not sure why I didn't just _search_ for it :sweat_smile: